core/src/main/scala/io/chrisdavenport/epimetheus/NameCommons.scala

```diff
@@ -48,11 +68,12 @@ trait NameCommons {
 
     def impl(s: String): Either[IllegalArgumentException, Suffix] = s match {
       case sufreg(string) => Either.right(new Suffix(string))
-      case _ => Either.left(
-        new IllegalArgumentException(
-          s"Input String - $s does not match regex - ([a-zA-Z0-9_:]*)"
+      case _ =>
```

Comment:
would it be possible to remove this whitespace change?

Reply:
fixed